### ISSUE TITLE
SP Alphaville - Cientista de Dados Sênior - ASSESSO INFORMÁTICA

### ISSUE THREAD

**Issue by RecrutamentoAssesso** [datascience-br/vagas#126]
## Descrição da vaga

- Realizar pesquisas, encontra insights e propõe soluções de Data Science/ Machine Learning, de acordo com a necessidade do cliente;
- Participa de discussões e desenvolvimento de PoCs (Provas de Conceito);
- Interação com o time de desenvolvedores para a integração de modelos de Machine Learning com o DataCare.

## Local

Híbrido: 3x/ semana - Centro Comercial Alphaville - Barueri/ SP

## O que oferecemos

- Remuneração: a combinar + Gympass, e parceria com workplace de seguros.
- Tipo de Contrato: PJ
- Modelo de trabalho: híbrido (3x/ semana)
- Local: Centro Comercial de Alphaville – Barueri/ SP

#### Obrigatórios:

- Experiência em ferramentas de Machine Learning 
- Experiência na manipulação e extração informações de repositórios de dados estruturados e não-estruturados
- Experiência em análise exploratória (capacidade de analisar um dataset e apresentar hipóteses)
- Experiência na utilização de bibliotecas de Machine Learning em Python / PySpark
- Experiência em técnicas de modelagem preditiva e descritiva
- Conhecimentos de estatística
- Conhecimento em bancos de dados diversos: Bancos de dados SQL e NoSQL
- Experiência no uso de bibliotecas Open source para análise, exploração e uso de modelos preditivos
- Experiência no uso de ferramentas em plataformas clouds (Google, Azure e AWS)

#### Diferenciais:

- Experiência com ferramentas de Gradient Boosting. (XGBoost, LGBM, etc)
- Experiência com TensorFlow
- Experiência com NLP
- Experiência com AWS Sagemaker pipeline

## Contratação

- PJ a combinar

## Nossa empresa

🏢 Sobre a empresa
A Assesso é líder no segmento de Qualidade e Governança de Dados, possui certificação do MIT e uma ferramenta própria chamada Datacare, que faz a gestão de dados dos nossos clientes. A nossa missão é ajudar o cliente a obter as informações assertivas e exatas, incluindo o enriquecimento com dados, estudos de probabilidades e tendências que eles possam necessitar. Para mais detalhes, acesse: assesso.com.br

## Como se candidatar

Por favor envie um email para rh@assesso.com.br com seu CV anexado - enviar no assunto: Cientista de Dados

#### Alocação
- Hibrido

#### Regime
- PJ

#### Nível
- Pleno
- Sênior
